yarn-project/pxe/src/entrypoints/client/bundle/utils.ts

```diff
@@ -0,0 +1,59 @@
+import { BBWASMBundlePrivateKernelProver } from '@aztec/bb-prover/wasm/bundle';
+import { createLogger } from '@aztec/foundation/log';
+import { createStore } from '@aztec/kv-store/indexeddb';
+import { BundledProtocolContractsProvider } from '@aztec/protocol-contracts/providers/bundle';
+import { WASMSimulator } from '@aztec/simulator/client';
+import type { AztecNode } from '@aztec/stdlib/interfaces/client';
+
+import type { PXEServiceConfig } from '../../../config/index.js';
+import { PXEService } from '../../../pxe_service/pxe_service.js';
+import type { PXECreationOptions } from '../pxe_creation_options.js';
+
+/**
+ * Create and start an PXEService instance with the given AztecNode.
+ * If no keyStore or database is provided, it will use KeyStore and MemoryDB as default values.
+ * Returns a Promise that resolves to the started PXEService instance.
+ *
+ * @param aztecNode - The AztecNode instance to be used by the server.
+ * @param config - The PXE Service Config to use
+ * @param options - (Optional) Optional information for creating an PXEService.
+ * @param proofCreator - An optional proof creator to use in place of any other configuration
+ * @returns A Promise that resolves to the started PXEService instance.
+ */
+export async function createPXEService(
```

Comment:
What does the 'bundle' dir mean?

Reply:
This is specifically for people creating service workers or any other environment that doesn't support lazy loading